src/classes/Commands.ts

```diff
@@ -582,6 +582,7 @@ export = class Commands {
             return;
         }
 
+        message = removeLinkProtocol(message);
         const paramStr = CommandParser.removeCommand(message);
```

Comment:
Instead of overwriting the message, just add do it in the argument.

```js
const paramStr = CommandParser.removeCommand(removeLinkProtocol(message));
```

Reply:
I see. alright 👌